sdk/keyvault/keyvault-certificates/src/transformations.ts

```diff
@@ -344,7 +344,11 @@ export function getCertificateOperationFromCoreOperation(
     certificateType: operation.issuerParameters
       ? operation.issuerParameters.certificateType
       : undefined,
-    csr: operation.csr,
+    // behavioral breaking change of onResponse callback
```

Comment:
The "parsedBody" from the onResponse callback used to be the body having been run through all the HLC deserializer logic but now it's just the result of calling `JSON.parse` on the raw response body. This is behavioral breaking change between the two Core versions but there's not much we can do about it.